wsdl/ver10/schema/onvif.xsd

```diff
@@ -3828,6 +3828,22 @@ decoding .A decoder shall decode every data it receives (according to its capabi
 		</xs:restriction>
 	</xs:simpleType>
 	<!--===============================-->
+	<xs:complexType name="EditableUserLevel">
+		<xs:sequence>
+			<xs:element name="Name" type="xs:string">
+				<xs:annotation>
+					<xs:documentation>Name of the editable user level.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="Functions" type="tt:StringList">
```

Comment:
Should we consider more compact data representation? For example grouping methods by namespace:
- namespace 1
  * function 1
  * function 2
  * function 3
-namespace 2
 * function 1
 * function 2